great_expectations/datasource/fluent/fluent_base_model.py

```diff
@@ -294,3 +301,28 @@ def _recursively_set_config_value(
                 data[i] = v.get_config_value(config_provider)
             elif isinstance(v, (MutableMapping, MutableSequence)):
                 return _recursively_set_config_value(v, config_provider)
+
+
+def _update__fields_set__on_truthyness(model: FluentBaseModel, field_name: str) -> None:
```

Comment:
@Kilo59 Do we need the double underscore between `set` and `on` in this method's name?  Thanks.

Reply:
Yes

Reply:
@Kilo59 Good point -- thank you.